share/locale/en.json

```diff
@@ -1,165 +1,165 @@
 {
-    "config_onion_service": "Configuring onion service on port {0:d}.",
-    "preparing_files": "Preparing files to share.",
+    "config_onion_service": "Setting up onion service on port {0:d}.",
+    "preparing_files": "Preparing OnionShare.",
     "wait_for_hs": "Waiting for HS to be ready:",
     "wait_for_hs_trying": "Trying…",
     "wait_for_hs_nope": "Not ready yet.",
-    "wait_for_hs_yup": "Ready!",
-    "give_this_url": "Give this address to the person you're sending the file to:",
-    "give_this_url_stealth": "Give this address and HidServAuth line to the person you're sending the file to:",
-    "give_this_url_receive": "Give this address to the people sending you files:",
-    "give_this_url_receive_stealth": "Give this address and HidServAuth line to the people sending you files:",
+    "wait_for_hs_yup": "Ready.",
+    "give_this_url": "Give this address to the recipient of the file:",
+    "give_this_url_stealth": "Give this address and HidServAuth line to the recipient of the file:",
+    "give_this_url_receive": "Give this address to those that send you files:",
+    "give_this_url_receive_stealth": "Give this address and your HidServAuth line to those that send you files:",
```

Comment:
"the that you want to send you files" again?

Reply:
Same problem as discussed above, but I think this explains the addition of the HidServAuth:
"Give this address and HidServAuth to people wishing to send you an OnionShare in a stealthy manner."